lib/amazon-pricing/ec2-dhi-price-list.rb

```diff
@@ -19,15 +19,18 @@ def os_types
     @@CAPACITY_HASH = {
       'c3' => { "large"=>16, "xlarge"=>8, "2xlarge"=>4, "4xlarge"=>2, "8xlarge"=>1 },
       'c4' => { "large"=>16, "xlarge"=>8, "2xlarge"=>4, "4xlarge"=>2, "8xlarge"=>1 },
+      'c5' => { "large"=>36, "xlarge"=>18, "2xlarge"=>9, "4xlarge"=>4,"9xlarge"=>2, "18xlarge"=>1 },
       'p2' => { "xlarge"=>16, "8xlarge"=>2, "16xlarge"=>1 },
       'p3' => { "2xlarge"=>8, "8xlarge"=>2, "16xlarge"=>1 },
       'g2' => { "2xlarge"=>4, "8xlarge"=>1 },
       'g3' => { "4xlarge"=>4, "8xlarge"=>2, "16xlarge"=>1 },
       'm3' => { "medium"=>32, "large"=>16, "xlarge"=>8, "2xlarge"=>4 },
       'd2' => { "xlarge"=>8, "2xlarge"=>4, "4xlarge"=>2, "8xlarge"=>1 },
+      'h1' => { "2xlarge"=>8, "4xlarge"=>4, "8xlarge"=>2, "16xlarge"=>1 },
```

Comment:
just curious, why does capacity goes down by size?

Reply:
it's how many dedicated instances can run for a given DH.